# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332116384 18000
#      Sun Mar 18 19:19:44 2012 -0500
# Node ID 91baffc7e83cf901cb335d75fdd8357d58548098
# Parent  4d9a75647d775ec4146b3847f2ba66073a09b62d
cmake: add install_name for shared prefix builds

diff -r 4d9a75647d77 -r 91baffc7e83c CMakeLists.txt
--- a/CMakeLists.txt	Sat Mar 30 16:08:12 2013 +0000
+++ b/CMakeLists.txt	Sun Mar 18 19:19:44 2012 -0500
@@ -17,10 +17,11 @@ execute_process(COMMAND ${CMAKE_COMMAND}
 # set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${MPI_COMPILE_FLAGS}")
 
 # Prepare libraries.
 if(SHARED)
   set(ParMETIS_LIBRARY_TYPE SHARED)
+  set(CMAKE_INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/lib")
 else()
   set(ParMETIS_LIBRARY_TYPE STATIC)
 endif()
 
 include(${GKLIB_PATH}/GKlibSystem.cmake)
